### DISCUSSION TITLE
Public sector only?

### DISCUSSION THREAD

**Discussion by akelew** [microsoft/PubSec-Info-Assistant#663] [Q&A]
Hi, I am looking to implement this as a proof of concept at an engineering and construction firm starting with internal policies. I only just noticed what the PubSec stands for.. Is there anything about this solution that would not be suitable for private sector? I'm struggling to understand what in the solution would be tailored towards specifically public sector. Thank you

**Comment by davidwaxman3283** [comment]
The Public Sector mention revolves around requirements that were driven because of the needs of Public Sector.   Specifically a focus on accuracy, and the ability to get a specific page when getting a reference.   We will also be adding a "secure deployment" option in the Hotfix which will come out in about 2 weeks.   The secure deployment option would give you the option of having a pre cyber hardened deployment.   Which is a long way around to saying it will work very well in the private sector as well.

**Reply by MetteNyland** [reply]
@davidwaxman3283 Will the "secure deployment" come out this week or when do you excpect it to be ready?

**Reply by davidwaxman3283** [reply]
@MetteNyland  It's taking longer than expected, but the good news is that once it's done you will be saving hundreds of hours.   I think we are still about 3 weeks away from releasing the secure version.

**Reply by ravikhunt** [reply]
Hi [davidwaxman3283](https://github.com/davidwaxman3283) @dayland is there any update on the timeline for secure deployment? Thank you!
